tests/integration/organizations_test.go

```diff
@@ -103,6 +103,27 @@ func TestOrganizations(t *testing.T) {
 	assert.NotEmpty(t, newOrganization2.ID)
 	assert.Equal(t, "my-org-2", newOrganization2.Name)
 
+	// Should short  Organizations by name DESC
+	orderBy := "-name"
+	organizations, err = client.Organizations().ListAll(clerk.ListAllOrganizationsParams{
+		OrderBy: &orderBy,
+	})
+
+	orderBy = "+name"
+	// Should short  Organizations by name ASC
+	organizations, err = client.Organizations().ListAll(clerk.ListAllOrganizationsParams{
+		OrderBy: &orderBy,
+	})
+
+	assert.Equal(t, organizations.Data[0].ID, newOrganization.ID)
+
+	if err != nil {
+		t.Fatalf("Organizations.ListAll order by name returned error: %v", err)
+	}
+	if organizations == nil {
+		t.Fatalf("Organizations.ListAll order by name returned nil")
+	}
```

Comment:
🙃 Should use use assertions instead of `if`s?
```
assert.Nil(t, err, "Organizations.ListAll order by name returned error");
assert.NotNil(t, organization, "Organizations.ListAll order by name returned nil");
```

Reply:
There's also 
```go
assert.NoError(t, err)
```
for errors.